waspc/examples/todoApp/src/user.ts

```diff
@@ -16,6 +16,11 @@ export function getName(user?: User) {
     return `Google user ${googleIdentity.providerUserId}`
   }
 
+  const githubIdentity = findUserIdentity(user, 'github')
+  if (githubIdentity) {
```

Comment:
> Still, since we're talking about a public API, we should probably do the ceremony and return null. Not now, of course, it's a breaking change that has nothing to do with this PR. But, if you agree, we can create an issue for polishing the public API.

Could you be more specific or better even, create the issue and link it here?

> NITPICK: In any case, I recommend using explicit comparisons instead of relying on coercion. I know typescript takes care of it, but I find it easier to verify the correctness of something like if (foo !== undefined) than if (foo).

Makes sense, I'll modify the helper.